lib/repository.js

```diff
@@ -41,11 +41,27 @@ async function getUserInformation(uid){
     return userDoc.data();
 }
 
+async function addUserComment(comment){
+    await addDoc(collection(db,'comments/' + comment.movieId + '/movieComments'),comment);
+    return comment;
+}
+
+async function getMovieComments(movieId){
+    const snapshot = await getDocs(query(collection(db, 'comments/' + movieId + '/movieComments'), where("movieId", "==", movieId)));
+    const commentsList = [];
+    snapshot.forEach((doc) => {
+        commentsList.push(doc.data());
+    })
+    return commentsList;
```

Comment:
You can probably do this with the `.map(...)` function if you want to practice some functional programming.